### DISCUSSION TITLE
习惯成就高效能 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#230] [Announcements]
https://mazhuang.org/2014/10/28/sevenhabits/

**Comment by 15311110209** [comment]
再次受益于博主

**Comment by mzlogin** [comment]
> @15311110209
> 再次受益于博主
> 

Glad to hear that~ :smile:

**Comment by 15311110209** [comment]
博主，向您请教一下关于如何配置项目的gitalk模块。有空的话可以教授一下吗？


在2017年10月19日 09:51，Zhuang Ma<notifications@github.com> 写道：

@15311110209
再次受益于博主

Glad to hear that~ 😄

**Comment by mzlogin** [comment]
> @15311110209
> 博主，向您请教一下关于如何配置项目的gitalk模块。有空的话可以教授一下吗？
> 
> 
> 在2017年10月19日 09:51，Zhuang Ma<notifications@github.com> 写道：
> 
> @15311110209
> 再次受益于博主
> 
> Glad to hear that~ 😄
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.

https://github.com/gitalk/gitalk/blob/master/readme-cn.md

官方文档我觉得讲得还比较清晰，你先照着弄一下试试~你可以的。
